### ISSUE TITLE
Lantern is down in China

### ISSUE THREAD

**Issue by HanWenfang** [getlantern/lantern#3107]
Chrome console outputs:

app.js:110 New websocket instance created [object Event] 
app.js:760 Got Lantern default settings:  Object
app.js:900 SENDING PAGE VIEW
app.js:787 Initializing proxied sites entries Array[7286]

http://127.0.0.1:16823/style/font/ProximaNova-Light.otf Failed to load resource: the server responded with a status of 404 (Not Found)

lib-ce052d1e02.js:17 Time since last connect 10954

app.js:277 Backend is gone

https://connect.facebook.net/en_US/sdk.js#appId=1562164690714282&xfbml=1&version=v2.3 Failed to load resource: net::ERR_TIMED_OUT

https://www.google-analytics.com/r/collect?v=1&_v=j38&aip=1&a=44636571&t=pa…d=1391665804&cid=888740182.1441512310&tid=UA-21815217-13&_r=1&z=1658634844 Failed to load resource: net::ERR_TIMED_OUT

**Comment by XhstormR** [comment]
me too,in china.
````
lantern.log
Sep 06 05:47:41.894 - 7m49s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 178.62.209.215:443 to tcp://logs-01.loggly.com:443: Unable to dial server (trusted) chained proxy at 178.62.209.215:443: dial tcp 178.62.209.215:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 11...continuing
Sep 06 05:47:41.923 - 7m49s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 128.199.41.202:443 to tcp://logs-01.loggly.com:443: Unable to dial server (trusted) chained proxy at 128.199.41.202:443: dial tcp 128.199.41.202:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 0...continuing
Sep 06 05:47:52.907 - 8m0s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 188.166.41.178:443 to tcp://clients4.google.com:443: Unable to dial server (trusted) chained proxy at 188.166.41.178:443: dial tcp 188.166.41.178:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 2...continuing
Sep 06 05:47:56.941 - 8m4s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 128.199.63.206:443 to tcp://logs-01.loggly.com:443: Unable to dial server (trusted) chained proxy at 128.199.63.206:443: dial tcp 128.199.63.206:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 0...continuing
Sep 06 05:47:59.909 - 8m7s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 128.199.63.206:443 to tcp://logs-01.loggly.com:443: Unable to dial server (trusted) chained proxy at 128.199.63.206:443: dial tcp 128.199.63.206:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 3...continuing
Sep 06 05:48:02.921 - 8m10s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 178.62.196.88:443 to tcp://logs-01.loggly.com:443: Unable to dial server (trusted) chained proxy at 178.62.196.88:443: dial tcp 178.62.196.88:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 1...continuing
Sep 06 05:48:10.131 - 8m17s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 188.166.41.178:443 to tcp://clients4.google.com:443: Unable to dial server (trusted) chained proxy at 188.166.41.178:443: dial tcp 188.166.41.178:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 0...continuing
Sep 06 05:48:11.939 - 8m19s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 178.62.247.181:443 to tcp://logs-01.loggly.com:443: Unable to dial server (trusted) chained proxy at 178.62.247.181:443: dial tcp 178.62.247.181:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 0...continuing
Sep 06 05:48:17.945 - 8m25s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 178.62.253.155:443 to tcp://logs-01.loggly.com:443: Unable to dial server (trusted) chained proxy at 178.62.253.155:443: dial tcp 178.62.253.155:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 1...continuing
Sep 06 05:48:20.914 - 8m28s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 178.62.209.215:443 to tcp://logs-01.loggly.com:443: Unable to dial server (trusted) chained proxy at 178.62.209.215:443: dial tcp 178.62.209.215:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 4...continuing
Sep 06 05:48:26.960 - 8m34s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 128.199.61.178:443 to tcp://logs-01.loggly.com:443: Unable to dial server (trusted) chained proxy at 128.199.61.178:443: dial tcp 128.199.61.178:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 0...continuing
Sep 06 05:48:31.129 - 8m38s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 128.199.44.168:443 to tcp://clients4.google.com:443: Unable to dial server (trusted) chained proxy at 128.199.44.168:443: dial tcp 128.199.44.168:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 1...continuing
Sep 06 05:48:38.946 - 8m46s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 178.62.196.88:443 to tcp://logs-01.loggly.com:443: Unable to dial server (trusted) chained proxy at 178.62.196.88:443: dial tcp 178.62.196.88:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 2...continuing
Sep 06 05:48:41.918 - 8m49s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 188.166.28.27:443 to tcp://logs-01.loggly.com:443: Unable to dial server (trusted) chained proxy at 188.166.28.27:443: dial tcp 188.166.28.27:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 5...continuing
Sep 06 05:48:41.957 - 8m49s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 178.62.231.213:443 to tcp://logs-01.loggly.com:443: Unable to dial server (trusted) chained proxy at 178.62.231.213:443: dial tcp 178.62.231.213:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 0...continuing
Sep 06 05:48:52.130 - 8m59s ERROR balancer: balancer.go:100 Unable to dial via (trusted) chained proxy at 178.62.249.247:443 to tcp://clients4.google.com:443: Unable to dial server (trusted) chained proxy at 178.62.249.247:443: dial tcp 178.62.249.247:443: ConnectEx tcp: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. on pass 2...continuing

````

**Comment by fishg** [comment]
Me too

**Comment by kingFighter** [comment]
Me too

**Comment by daerdai** [comment]
Me too!!

**Comment by fhzheng** [comment]
Me too

**Comment by ustcscgy** [comment]
Please, people, at least report your lantern version and operating system version, and a short description of your issue. @fishg  @kingFighter  @daerdai  @fhzheng 

I use lantern 2.0.3 (from branch release-2.0.1, tag 2.0.3, commit 313cc6d3ffe602d) compiled from source on Linux, and have no problem. Lantern is up and fast here.

**Comment by fishg** [comment]
@ustcscgy My version is 2.0.3 for OS X,download from getlantern.org today.My system is OS X 10.11. 
But I guess the problem is about China's network limit,not the soft bug.Attach part of my log:
```
DEBUG enproxy: conn_intf.go:239 Failing on Bad response status for read from fronting provider: HTTP/1.1 400 Bad Request
Connection: close
Content-Length: 177
Content-Type: text/html
Date: Sun, 06 Sep 2015 02:49:37 GMT
Server: cloudflare-nginx

<html>
<head><title>400 Bad Request</title></head>
<body bgcolor="white">
<center><h1>400 Bad Request</h1></center>
<hr><center>cloudflare-nginx</center>
</body>
</html>

Sep 06 02:49:37.662 - 0m30s ERROR fronted: masquerade.go:169 HTTP error for masquerade www.cozi.com: Head http://www.google.com/humans.txt: Dest: www.google.com:80    ProxyHost:     Unable to issue write request: Bad response status for read from fronting provider: HTTP/1.1 400 Bad Request
Connection: close
Content-Length: 177
Content-Type: text/html
Date: Sun, 06 Sep 2015 02:49:37 GMT
Server: cloudflare-nginx

<html>
<head><title>400 Bad Request</title></head>
<body bgcolor="white">
<center><h1>400 Bad Request</h1></center>
<hr><center>cloudflare-nginx</center>
</body>
</html>

Sep 06 02:49:37.687 - 0m30s DEBUG enproxy: conn_impl_requests.go:53 Issued write request with result: Bad response status for read from fronting provider: HTTP/1.1 400 Bad Request
Connection: close
Content-Length: 177
Content-Type: text/html
Date: Sun, 06 Sep 2015 02:49:37 GMT
Server: cloudflare-nginx

<html>
<head><title>400 Bad Request</title></head>
<body bgcolor="white">
<center><h1>400 Bad Request</h1></center>
<hr><center>cloudflare-nginx</center>
</body>
</html>

Sep 06 02:49:37.687 - 0m30s DEBUG enproxy: conn_intf.go:239 Failing on Dest: www.google.com:80    ProxyHost:     Unable to issue write request: Bad response status for read from fronting provider: HTTP/1.1 400 Bad Request
Connection: close
Content-Length: 177
Content-Type: text/html
Date: Sun, 06 Sep 2015 02:49:37 GMT
Server: cloudflare-nginx

<html>
<head><title>400 Bad Request</title></head>
<body bgcolor="white">
<center><h1>400 Bad Request</h1></center>
<hr><center>cloudflare-nginx</center>
</body>
</html>

Sep 06 02:49:37.687 - 0m30s DEBUG enproxy: conn_intf.go:239 Failing on Bad response status for read from fronting provider: HTTP/1.1 400 Bad Request
Connection: close
Content-Length: 177
Content-Type: text/html
Date: Sun, 06 Sep 2015 02:49:37 GMT
Server: cloudflare-nginx

<html>
<head><title>400 Bad Request</title></head>
<body bgcolor="white">
<center><h1>400 Bad Request</h1></center>
<hr><center>cloudflare-nginx</center>
</body>
</html>
```

```
Sep 06 12:06:26.607 - 0m13s DEBUG balancer: balancer.go:96 Dialing tcp://logs-01.loggly.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:06:30.463 - 0m17s DEBUG enproxy: conn_impl.go:46 Proxy connection to www.google.com:80 via 67.225.226.131:443 idle for 10s, closing
Sep 06 12:06:49.400 - 0m36s ERROR fronted: masquerade.go:169 Timed out verifying imsuccesscenter.com
Sep 06 12:06:50.075 - 0m37s DEBUG balancer: balancer.go:96 Dialing tcp://notify9.dropbox.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:06:53.234 - 0m40s DEBUG balancer: balancer.go:96 Dialing tcp://logs-01.loggly.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:06:53.613 - 0m40s DEBUG flashlight.ui: ws.go:156 Error reading from UI: websocket: close 1005 
Sep 06 12:06:53.614 - 0m40s DEBUG balancer: balancer.go:96 Dialing tcp://www.google.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:06:53.615 - 0m40s DEBUG balancer: balancer.go:96 Dialing tcp://ssl.gstatic.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:13.851 - 1m0s DEBUG balancer: balancer.go:96 Dialing tcp://twitter.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:13.851 - 1m0s DEBUG balancer: balancer.go:96 Dialing tcp://abs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:13.851 - 1m0s DEBUG balancer: balancer.go:96 Dialing tcp://abs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:13.851 - 1m0s DEBUG balancer: balancer.go:96 Dialing tcp://abs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:13.852 - 1m0s DEBUG balancer: balancer.go:96 Dialing tcp://abs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:13.852 - 1m0s DEBUG balancer: balancer.go:96 Dialing tcp://abs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:13.852 - 1m0s DEBUG balancer: balancer.go:96 Dialing tcp://abs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:13.852 - 1m0s DEBUG balancer: balancer.go:96 Dialing tcp://pbs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:13.852 - 1m0s DEBUG balancer: balancer.go:96 Dialing tcp://pbs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:13.852 - 1m0s DEBUG balancer: balancer.go:96 Dialing tcp://pbs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:13.853 - 1m0s DEBUG balancer: balancer.go:96 Dialing tcp://pbs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:13.853 - 1m0s DEBUG balancer: balancer.go:96 Dialing tcp://pbs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:13.853 - 1m0s DEBUG balancer: balancer.go:96 Dialing tcp://pbs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:43.956 - 1m31s DEBUG balancer: balancer.go:96 Dialing tcp://twitter.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:48.822 - 1m35s DEBUG balancer: balancer.go:96 Dialing tcp://ssl.gstatic.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:07:57.528 - 1m44s DEBUG detour: detour.go:178 Error while read from updates.fournova.com:443 initially: read tcp 54.172.108.3:443: i/o timeout
Sep 06 12:07:57.528 - 1m44s DEBUG detour: detour.go:183 Detour HTTP GET request to updates.fournova.com:443
Sep 06 12:07:57.528 - 1m44s DEBUG balancer: balancer.go:96 Dialing tcp://updates.fournova.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:08:13.977 - 2m1s DEBUG balancer: balancer.go:96 Dialing tcp://www.gstatic.com:80 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:08:18.968 - 2m6s DEBUG balancer: balancer.go:96 Dialing tcp://twitter.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:08:20.089 - 2m7s DEBUG balancer: balancer.go:96 Dialing tcp://notify9.dropbox.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:08:22.171 - 2m9s DEBUG enproxy: conn_impl_requests.go:53 Issued write request with result: Error reading response from proxy: unexpected EOF
Sep 06 12:08:22.171 - 2m9s DEBUG enproxy: conn_intf.go:239 Failing on Dest: www.google.com:80    ProxyHost:     Unable to issue write request: Error reading response from proxy: unexpected EOF
Sep 06 12:08:22.172 - 2m9s DEBUG enproxy: conn_intf.go:239 Failing on Error reading response from proxy: unexpected EOF
Sep 06 12:08:48.834 - 2m35s DEBUG balancer: balancer.go:96 Dialing tcp://ssl.gstatic.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:08:59.739 - 2m46s DEBUG balancer: balancer.go:96 Dialing tcp://notify9.dropbox.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:06.172 - 2m53s DEBUG balancer: balancer.go:96 Dialing tcp://dl-debug104.dropbox.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:11.398 - 2m58s DEBUG balancer: balancer.go:96 Dialing tcp://twitter.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:11.398 - 2m58s DEBUG balancer: balancer.go:96 Dialing tcp://abs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:11.398 - 2m58s DEBUG balancer: balancer.go:96 Dialing tcp://abs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:11.399 - 2m58s DEBUG balancer: balancer.go:96 Dialing tcp://abs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:11.399 - 2m58s DEBUG balancer: balancer.go:96 Dialing tcp://pbs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:11.399 - 2m58s DEBUG balancer: balancer.go:96 Dialing tcp://pbs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:11.399 - 2m58s DEBUG balancer: balancer.go:96 Dialing tcp://abs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:11.399 - 2m58s DEBUG balancer: balancer.go:96 Dialing tcp://pbs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:11.399 - 2m58s DEBUG balancer: balancer.go:96 Dialing tcp://pbs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:11.399 - 2m58s DEBUG balancer: balancer.go:96 Dialing tcp://pbs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:11.400 - 2m58s DEBUG balancer: balancer.go:96 Dialing tcp://pbs.twimg.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:19.359 - 3m6s DEBUG balancer: balancer.go:96 Dialing tcp://notify9.dropbox.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:19.552 - 3m6s DEBUG balancer: balancer.go:96 Dialing tcp://notify9.dropbox.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:19.807 - 3m6s DEBUG detour: detour.go:175 Unable to set read deadline: use of closed network connection
Sep 06 12:09:19.807 - 3m6s DEBUG detour: detour.go:178 Error while read from configuration.apple.com:443 initially: read tcp 184.29.58.159:443: use of closed network connection
Sep 06 12:09:19.856 - 3m6s DEBUG balancer: balancer.go:96 Dialing tcp://twitter.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:21.615 - 3m8s DEBUG detour: detour.go:138 Dial initially to www.stackvdi.com:80 failed, try detour: dial tcp 188.226.168.210:80: i/o timeout
Sep 06 12:09:21.615 - 3m8s DEBUG balancer: balancer.go:96 Dialing tcp://www.stackvdi.com:80 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:22.602 - 3m9s DEBUG detour: detour.go:138 Dial initially to www.stackvdi.com:80 failed, try detour: dial tcp 188.226.168.210:80: i/o timeout
Sep 06 12:09:22.602 - 3m9s DEBUG balancer: balancer.go:96 Dialing tcp://www.stackvdi.com:80 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:50.264 - 3m37s DEBUG balancer: balancer.go:96 Dialing tcp://notify9.dropbox.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:54.502 - 3m41s DEBUG detour: detour.go:178 Error while read from blog.xe28.tk:80 initially: read tcp 195.20.42.17:80: i/o timeout
Sep 06 12:09:54.502 - 3m41s DEBUG detour: detour.go:183 Detour HTTP GET request to blog.xe28.tk:80
Sep 06 12:09:54.502 - 3m41s DEBUG balancer: balancer.go:96 Dialing tcp://blog.xe28.tk:80 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:54.777 - 3m41s DEBUG detour: detour.go:138 Dial initially to furbo.org:80 failed, try detour: dial tcp 38.103.165.50:80: i/o timeout
Sep 06 12:09:54.777 - 3m41s DEBUG balancer: balancer.go:96 Dialing tcp://furbo.org:80 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:58.271 - 3m45s DEBUG balancer: balancer.go:96 Dialing tcp://notify9.dropbox.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:09:58.915 - 3m46s DEBUG balancer: balancer.go:96 Dialing tcp://notify9.dropbox.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:10:00.376 - 3m47s DEBUG detour: detour.go:134 Dial initially to blog.4321.la:80 failed: dial tcp: lookup blog.4321.la: no such host
Sep 06 12:10:00.396 - 3m47s DEBUG detour: detour.go:134 Dial initially to blog.4321.la:80 failed: dial tcp: lookup blog.4321.la: no such host
Sep 06 12:10:00.396 - 3m47s DEBUG detour: detour.go:134 Dial initially to blog.4321.la:80 failed: dial tcp: lookup blog.4321.la: no such host
Sep 06 12:10:00.405 - 3m47s DEBUG detour: detour.go:134 Dial initially to blog.4321.la:80 failed: dial tcp: lookup blog.4321.la: no such host
Sep 06 12:10:00.413 - 3m47s DEBUG detour: detour.go:134 Dial initially to blog.4321.la:80 failed: dial tcp: lookup blog.4321.la: no such host
Sep 06 12:10:00.423 - 3m47s DEBUG detour: detour.go:134 Dial initially to blog.4321.la:80 failed: dial tcp: lookup blog.4321.la: no such host
Sep 06 12:10:00.429 - 3m47s DEBUG detour: detour.go:134 Dial initially to blog.4321.la:80 failed: dial tcp: lookup blog.4321.la: no such host
Sep 06 12:10:01.269 - 3m48s DEBUG detour: detour.go:138 Dial initially to furbo.org:80 failed, try detour: dial tcp 38.103.165.50:80: i/o timeout
Sep 06 12:10:01.269 - 3m48s DEBUG balancer: balancer.go:96 Dialing tcp://furbo.org:80 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:10:02.149 - 3m49s DEBUG detour: detour.go:178 Error while read from blog.xe28.tk:80 initially: read tcp 195.20.42.17:80: i/o timeout
Sep 06 12:10:02.149 - 3m49s DEBUG detour: detour.go:183 Detour HTTP GET request to blog.xe28.tk:80
Sep 06 12:10:02.149 - 3m49s DEBUG balancer: balancer.go:96 Dialing tcp://blog.xe28.tk:80 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:10:03.519 - 3m50s DEBUG detour: detour.go:138 Dial initially to xuzonghao.cn:8090 failed, try detour: dial tcp 118.244.0.206:8090: i/o timeout
Sep 06 12:10:03.519 - 3m50s DEBUG balancer: balancer.go:96 Dialing tcp://xuzonghao.cn:8090 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:10:03.987 - 3m51s DEBUG detour: detour.go:138 Dial initially to rhel.ieesee.net:80 failed, try detour: dial tcp 61.155.95.79:80: i/o timeout
Sep 06 12:10:03.987 - 3m51s DEBUG balancer: balancer.go:96 Dialing tcp://rhel.ieesee.net:80 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:10:14.341 - 4m1s DEBUG balancer: balancer.go:96 Dialing tcp://notify9.dropbox.com:443 with (trusted) fronted proxy at nl.fallbacks.getiantem.org:443 using masquerade set cloudflare
Sep 06 12:10:15.337 - 4m2s DEBUG flashlight: flashlight.go:454 Calling exit func

```

**Comment by fishg** [comment]
Other wise nl.fallbacks.getiantem.org 's 443 port have certificate  error if use https.
```
curl https://nl.fallbacks.getiantem.org
curl: (51) SSL: certificate subject name 'ssl4742.cloudflare.com' does not match target host name 'nl.fallbacks.getiantem.org'
```

**Comment by ustcscgy** [comment]
@fishg  I have the exact same 400 Error at first 23 seconds, but after that lantern is normal. Also,

````
curl https://nl.fallbacks.getiantem.org
curl: (51) SSL: no alternative certificate subject name matches target host name 'nl.fallbacks.getiantem.org'
````
. but I can still connect, so I don't think that's the problem.

**Comment by fletchere** [comment]
me to, rolled back to 2.0.2, but 2.0.2 can not connect anymore.

**Comment by DavidSuccess** [comment]
The server automatically switch to the Netherlands，and very slow.

**Comment by dyis** [comment]
2.0.2 is slow and new beta is blocked

**Comment by myleshorton** [comment]
Hi everyone! Very sorry for the delay, but we've experienced a more general blocking event actually not related to the GFW but related to us using CloudFlare. We are currently working hard to resolve the issue and should have a fix available within the next 48 hours. Thanks for you patience @dyis, @DavidSuccess, @fletchere, @ustcscgy, @fishg, @daerdai, @kingFighter, @XhstormR, and @HanWenfang

We will be back up soon.

**Comment by HanWenfang** [comment]
@myleshorton Thanks for your excellent work!

**Comment by kingFighter** [comment]
@myleshorton Thanks for your excellent work!

**Comment by johnwatsondev** [comment]
@myleshorton We want to donate this great library! 
@HanWenfang @kingFighter  What about this idea you guys?

**Comment by myleshorton** [comment]
Thanks @HanWenfang, @kingFighter, and @johnwatsondev! We'll also likely release a Lantern Pro version that will be a little more like Shadowsocks -- it will basically allow you to pay for your own private server you can share with your friends. Unlike Shadowsocks, though, it will all be automated.

So it will be more resistant to blocking in several ways, and it should also generally be faster. It won't be available right away, but quite soon! And it will be very reasonably priced.

**Comment by YDSS** [comment]
@myleshorton It'll be very helpful, so where should I download this ** Pro version**

**Comment by fletchere** [comment]
@YDSS stay tuned, @myleshorton said it won't be availble right away,hehe.

**Comment by jxmlhdf** [comment]
@myleshorton I don't think that develop A shadow-liked version of Lantern is wise decision. Is there any pure free things left in this world? Is that violate the initial principle of Lantern to develop a paid version and called lantern Pro??? why don't you guys spend time to solve the issues effectively rather than looking into some stupid things??

**Comment by ayyb1988** [comment]
@myleshorton  Thanks very much,but where i can find Lantern Pro version

**Comment by XhstormR** [comment]
@ayyb1988 It won't be available right away, but quite soon!

**Comment by dozer47528** [comment]
2.0.4 still down in China.

I'm using mac os 10.11,where can I find the log file?

**Comment by myleshorton** [comment]
Thanks @dozer47528 your log file should be at:

/Users/YOUR_USERNAME/Library/Logs/Lantern/lantern.go

Can you send it to myles@getlantern.org?

What happens when you run it?

**Comment by dozer47528** [comment]
@myleshorton 
I have send my logs to your email.

**Comment by jxmlhdf** [comment]
Stupid updates which even took 5 days !!

**Comment by johnwatsondev** [comment]
@jxmlhdf Please pull your request or log file other than just blame it.

**Comment by Centaur** [comment]
FYI.  2.0.2 still works here, China Telecom Shanghai,  connected to JP server, acceptable speed.  OS X.

**Comment by myleshorton** [comment]
Awesome @Centaur thanks. We're working on changes now that should essentially pick up your 2.0.2 working config and will keep using it. Basically the problem is the CloudFlare stopped supporting domain fronting, and we were relying on that, so 2.0.2 should still work no problem but won't do things like automatically fetch updated configurations because of the CloudFlare issue. The new version will work around that but won't be out for another day or so.

**Comment by myleshorton** [comment]
@fishg @kingFighter @daerdai @fhzheng @johnwatsondev @Centaur we just released 2.0.6, which should work much better for you.

**Comment by fishg** [comment]
@myleshorton Great.New version is work fine.
